connector/mqtt/vernemq/k2v-bridge/bin/scripts_tls/signCert.sh

```diff
@@ -10,15 +10,15 @@ BASE_DIR=${BASE_DIR:-"/k2v_bridge"}
 
 . "${BASE_DIR}"/bin/scripts_tls/saveFormattedCRT.sh
 
-echo "Signing cert for entity ${certCname} in ${certCAName} : ${certEjbcaApiUrl}/sign/${certCname}/pkcs10 "
-csrContent=$(<  "${certDir}"/"${certCsrFile}"  sed '1,1d;$ d' | tr -d '\r\n')
+echo
+echo "Signing cert : ${certEjbcaApiUrl}/internal/api/v1/throw-away "
 
-signCertCa=$(curl --silent -X POST "${certEjbcaApiUrl}"/sign/"${certCname}"/pkcs10 \
+csrContent=$(while IFS= read -r line; do printf '%s\\n' "$line"; done <"${certDir}/${certCsrFile}")
+csrContent=${csrContent:0:(-2)}
+
+signCertCa=$(curl -X POST "${certEjbcaApiUrl}"/internal/api/v1/throw-away \
 -H "Content-Type:application/json" \
```

Comment:
Make sense keep using the header `Content-Type:application/json`, in the previous implementation a JSON was used to build the request, but the new one does not.